src/Compilers/CSharp/Test/Syntax/Parsing/ExtensionParsingTests.cs

```diff
@@ -3513,37 +3040,21 @@ class extension
     public void ExplicitConversionOperator_ExplicitImplementationInExtensionNamespace()
     {
         var text = """
-class C : I<C>
-{
-    static explicit extension.I<C>.operator int(C i)
-    {
-        return 1;
-    }
-}
-""";
+            class C : I<C>
+            {
+                static explicit extension.I<C>.operator int(C i)
+                {
+                    return 1;
+                }
+            }
+            """;
         // PROTOTYPE consider avoiding this break (in following context)
         // namespace extension;
         // public interface I<T> where T : I<T>
         // {
         //    public abstract static explicit operator int(T i);
         // }
-        UsingTreeWithCSharpNext(text,
-            // (3,30): error CS1001: Identifier expected
-            //     static explicit extension.I<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_IdentifierExpected, ".").WithLocation(3, 30),
-            // (3,30): error CS1514: { expected
-            //     static explicit extension.I<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_LbraceExpected, ".").WithLocation(3, 30),
-            // (3,30): error CS1513: } expected
-            //     static explicit extension.I<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_RbraceExpected, ".").WithLocation(3, 30),
-            // (3,30): error CS1519: Invalid token '.' in class, record, struct, or interface member declaration
-            //     static explicit extension.I<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_InvalidMemberDecl, ".").WithArguments(".").WithLocation(3, 30),
-            // (3,31): error CS1003: Syntax error, 'explicit' expected
-            //     static explicit extension.I<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_SyntaxError, "I").WithArguments("explicit").WithLocation(3, 31)
-            );
+        UsingTreeWithCSharpNext(text);
```

Comment:
>UsingTreeWithCSharpNext(text); [](http://example.com/codeflow?start=8&length=30)

It looks like the comment above has been addressed

Reply:
done.